### ISSUE TITLE
Sidebar transparency and blur

### ISSUE THREAD

**Issue by adamgall** [decentdao/fractal-interface#1730]
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiY2NhZDFmZmUtZDU4ZS00MjM4LWJiY2EtY2MwOTMxNTg1NWNmIiwiZGVwbG95X2lkIjoiNjYzYjhmOWZjZDQ0NDQwMDA4MzIwNjYzIiwiaWQiOiI3NzQ0YjdlNS04NjI5LTQ3YzAtOGEyYy1iYmMwNDZhMTVmMjkifQ==)

Sidebar needs transaparency and background blur

<details><summary>Browser metadata</summary>

```
Path:      /
Browser:   Firefox 125.0 on Mac OS 10.15
Viewport:  1402 x 912 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1578.app.dev.fractalframework.xyz%2F&os=OS%20X&os_version=Catalina&browser=Firefox&browser_version=125.0&resolution=1402x912&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-1578.app.dev.fractalframework.xyz/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=663b8f9fcd44440008320663&commentID=663babe54962130082799864&resolution=resolved)

_Originally posted by @nicolaus-sherrill in https://github.com/decentdao/fractal-interface/issues/1578#issuecomment-2100990724_

**Comment by Da-Colon** [comment]
This is mentioned in #1684, but might be good to have a separate issue

**Comment by Da-Colon** [comment]
Is this what you are looking for? (or can you point me to the the right design? I removed the BG and added the blur and box-shadow.  @xraystyle1980 

![Image](https://github.com/decentdao/fractal-interface/assets/38386583/eb7893ce-9ff6-4b41-8e5f-46ee89012c7d)

**Comment by adamgall** [comment]
@Da-Colon I don't think we wanted full transparency here. The existing background color just needed like an alpha channel on it or something. I don't really know the right terminology to use. Paging @xraystyle1980 for the deets

**Comment by Da-Colon** [comment]
> @Da-Colon I don't think we wanted full transparency here. The existing background color just needed like an alpha channel on it or something. I don't really know the right terminology to use. Paging @xraystyle1980 for the deets

Thats probably correct. I thought about that after I posted the response that we could just use the same RGBA that is used for the main `Header` component. 

to that point (unrelated to this card) but what if we swapped how the Header component is blurred. as the problem current is that the header's children menu can't be blured but if we main the Header a solid color and just move the blur to the menus?

**Comment by adamgall** [comment]
Closed via https://github.com/decentdao/fractal-interface/pull/1782
